### PR TITLE
Update cordova wrapper instructions

### PR DESCRIPTION
**Pull request by spekulatius**
Hello Razvan,

I've expanded the instructions for the Cordova wrapper a bit and added the troubleshooting information from my recent forum post (http://forum.quasar-framework.org/topic/390/resolved-cordova-requirements-android-sdk-not-found-on-ubuntu-16-04-based-os) in. Please let me know if you think it misses anything.

Peter

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe:
 Documentation for Cordova wrapper enhanced.

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested with all major browsers

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by rstoenescu** [comment]
Thanks, Peter! Looking forward to merging this when it's ready.

**Comment by spekulatius** [comment]
So, I fixed the last typo I just noticed :D

**Comment by rstoenescu** [comment]
Is it ready now?

**Comment by spekulatius** [comment]
Yupp. Please have a look and let me know if you aren't happy with any aspect :)

P.s. I assume the images will work after the deployment.

File: source/guide/cordova-wrapper.md [hunk 1]
```diff
@@ -1,19 +1,45 @@
...
+	Select the desired SDKs. As per May 2017 Cordova supports 4.4 and up and click on "Apply" to install the SDKs.
+
+	![SDK selection](/images/Android-Studio-SDK-selection.png "SDK selection")
 
-## Wrap your App
-Now let's create the Wrapper for your App.
+
+## Wrapping and configuration of your App
+
+Each Quasar project requires you to wrap your app initially. The following steps explain how to wrap your app and add the desired platforms.
```
Comment (zuck) [inline source/guide/cordova-wrapper.md]: This sentence:

> Each Quasar project requires you to wrap your app initially.

is a little bit misleading...Maybe it's better to replace it with something like:

> To transform a Quasar project into a working Cordova app, you need to wrap it initially.

Comment (spekulatius) [inline source/guide/cordova-wrapper.md]: Yeah, I've fixed this to your suggestion @zuck

Comment (spekulatius) [inline source/guide/cordova-wrapper.md]: Yeah, I've fixed this to your suggestion @zuck

**Comment by rstoenescu** [comment]
Thanks all!

**Pull request merged**